pkg/virt-config/deprecation/macvtap.go

```diff
@@ -23,8 +23,14 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 )
 
+const MacvtapGate = "Macvtap" // Deprecated
+
 const MacvtapDiscontinueMessage = "Macvtap network binding is discontinued since v1.3. Please refer to Kubevirt user guide for alternatives."
 
+func init() {
+	RegisterFeatureGate(FeatureGate{Name: MacvtapGate, State: Discontinued, Message: MacvtapDiscontinueMessage, VmiSpecUsed: macvtapApiUsed})
+}
+
```

Comment:
In the PR description you wrote:
> New FGs are encourage to be registered from a location that is owned by the relevant feature owner.
> Existing FGs may be moved to the relevant feature owners.

Is this an example for that? Or have I misunderstood?

Reply:
Yes, but it is partial.
It shows that the two are not in the same pool as the others.

A follow up should move this to the sig-network sub-tree.

As this PR is large enough, I prefer to defer the second part, but if reviewers will insist, I will do it in this PR.

Reply:
Yeah that makes sense.
I think it'd be great from a readability perspective that the FGs would still live near to one another, and not completely spread around. But we can discuss that in your follow up :+1:

Reply:
I tried to explain the motivation of why decentralizing it is better in the prev [thread](https://github.com/kubevirt/kubevirt/pull/11989/commits/9a240c05aa3c1bbad353bf7624480f8d1c2061af#r1851544665).

The fact that the FGs are now in a container, allows us to report them in any way we want. E.g. through a CLI or through an export to a YAML during the build (i.e. generate it using `make generate`).
This can answer both needs and can be handled in a follow up.